test/unit/features/test_pagination_base.py

```diff
@@ -309,8 +310,11 @@ def test_as_pager_get_all(self):
       pager: Pager[ViewResultRow] = pagination.pager()
       actual_items = pager.get_all()
       self.assertSequenceEqual(actual_items, mock.all_expected_items(), "The results should match all the pages.")
+    # Assert that we are going to assert the correct msg
+    expected_msg = _IteratorPager._state_consumed_msg
+    self.assertIn('consumed', expected_msg)
```

Comment:
I think this is over-engineering, but understand why you double-check this here.

Can we check these only once instead of calling it in  each test?